$NetBSD: patch-retrie.cabal,v 1.1 2022/02/16 10:11:00 pho Exp $

* Fix build with ghc-exactprint-1.5
* Fix build with optparse-applicative-0.17
* Although we want the executable "retrie" to be built, we don't want
  "demo-retrie". The upstream should really decouple these.

--- retrie.cabal.orig	2001-09-09 01:46:40.000000000 +0000
+++ retrie.cabal
@@ -84,10 +84,10 @@ library
     directory >= 1.3.1 && < 1.4,
     filepath >= 1.4.2 && < 1.5,
     ghc >= 9.2,
-    ghc-exactprint >= 1.4.0 && < 1.5,
+    ghc-exactprint >= 1.4.0,
     list-t >= 1.0.4 && < 1.1,
     mtl >= 2.2.2 && < 2.3,
-    optparse-applicative >= 0.15.1 && < 0.17,
+    optparse-applicative >= 0.15.1,
     process >= 1.6.3 && < 1.7,
     random-shuffle >= 0.0.4 && < 0.1,
     syb >= 0.7.1 && < 0.8,
@@ -118,10 +118,7 @@ executable retrie
   default-language: Haskell2010
 
 executable demo-retrie
-  if flag(BuildExecutable)
-    Buildable: True
-  else
-    Buildable: False
+  Buildable: False
   main-is:
     Main.hs
   hs-source-dirs: demo hse
